### PR TITLE
Added OnceCell::into_inner() for sync and unsync.

### PR DESCRIPTION
**Pull request by manuthambi**
#28

### CODE REVIEW

File: src/imp_pl.rs [hunk 1]
```diff
@@ -87,7 +87,7 @@ impl<T> OnceCell<T> {
         }
     }
 
-    pub fn get_or_try_init<F: FnOnce() -> Result<T, E>, E>(&self, f: F) -> Result<&T, E> {
+    pub(crate) fn get_or_try_init<F: FnOnce() -> Result<T, E>, E>(&self, f: F) -> Result<&T, E> {
```
Comment (matklad) [inline src/imp_pl.rs]: both `get_or_try_init` and `into_inner` should be just `pub` to be visible from outside

Comment (matklad) [reply src/imp_pl.rs]: Ah, sorry, that's pl impl, it should be `pub(crate)` indeed, thanks!

**Comment by manuthambi** [comment]
Thanks for reviewing and merging it!

**Comment by matklad** [comment]
published as 0.2.2!

**Pull request merged**